plugin/advancedsubscription/ajax/advsub.ajax.php

```diff
@@ -0,0 +1,248 @@
+<?php
```

Comment:
The plugin method `get_name()` returns the plugin class name in lowercase (e.g advancedsubscription)
And this method is used by plugin method `get_lang()` use this to locate the lang folder, but the folder name (plugin name) is wrong.
This is used for plugin info and cant translate it (e.g from plugin list, it will show "plugin_title")
By the moment, override `get_name()` method for advanced subscription plugin